CHANGELOG.md

```diff
@@ -3,6 +3,66 @@
+ - **FEAT**(nextcloud): Implement WebDAV extended MKCOL. ([91752b3f](https://github.com/nextcloud/neon/commit/91752b3f36ca60ccbe4b4ecfa834067ed0b53616))
+ - **FEAT**(nextcloud): Add support for checksums for uploading WebDAV files. ([4380f5ad](https://github.com/nextcloud/neon/commit/4380f5ad268b2723ef885dc398b77285d33803fa))
+ - **DOCS**(nextcloud): Improve method docs of WebDavClient. ([a018df24](https://github.com/nextcloud/neon/commit/a018df247575985ee8b1d486e33d6a09cba2556b))
+ - **BREAKING** **FIX**(nextcloud): Move RichObjectParameter from spreed to core and fix rich parameter typing for broken apps. ([f4fd9cf8](https://github.com/nextcloud/neon/commit/f4fd9cf8c52a43c70d38e2a1572b2b34e6d04d5c))
+ - **BREAKING** **FIX**(nextcloud): Throw DynamiteStatusCodeExceptions for invalid WebDAV status codes. ([801599c5](https://github.com/nextcloud/neon/commit/801599c510041b1d3e142b4e7823aaf6cb6f1d09))
+ - **BREAKING** **FIX**(nextcloud): Make files_sharing Share label nullable. ([8e7a47dd](https://github.com/nextcloud/neon/commit/8e7a47dd437436f99175b406116b72cdd9b596cf))
+ - **BREAKING** **FIX**(nextcloud): Fix deleted parent messages in spreed. ([5f716d1d](https://github.com/nextcloud/neon/commit/5f716d1dc960b9aa6087a7cec2c6a0f7bf08ce0e))
+ - **BREAKING** **FEAT**(nextcloud): make WebdavClient a DynamiteClient. ([bc8b54c8](https://github.com/nextcloud/neon/commit/bc8b54c852b0ddb77d08384d50bc11b951566bb6))
+ - **BREAKING** **FEAT**(nextcloud): Update to tables 0.8. ([13c677d7](https://github.com/nextcloud/neon/commit/13c677d7b7efd13f2c140d6a92567db6675b0fa4))
+ - **BREAKING** **FEAT**(nextcloud): Update to spreed 20. ([afdb4c38](https://github.com/nextcloud/neon/commit/afdb4c38a48cb22264004805b66ef80e68da65d6))
+ - **BREAKING** **FEAT**(nextcloud): Update to notifications 30. ([d9795186](https://github.com/nextcloud/neon/commit/d9795186be62b4c6715c5663f8b8809bfda091a7))
+ - **BREAKING** **FEAT**(nextcloud): Update to server 30. ([16c6272b](https://github.com/nextcloud/neon/commit/16c6272b23cfd0241fa5b22762e0850c5525cd60))
+ - **BREAKING** **CHORE**(nextcloud): Remove deprecated userAgent and cookieJar parameters from NextcloudClient. ([4ce78c1f](https://github.com/nextcloud/neon/commit/4ce78c1f68ac1fb550ba4e41f3cdc6297ccccdc7))
+ - **BREAKING** **CHORE**(nextcloud): Remove deprecated AppIDs. ([ab2fb5a2](https://github.com/nextcloud/neon/commit/ab2fb5a2a1f36b7e5c50facb8b5699ae09185315))
+
+#### `dynamite` - `v0.5.0+1`
+
+ - **REFACTOR**: change package file structure. ([6696e6ea](https://github.com/nextcloud/neon/commit/6696e6ead1751462e233afb5eab22a458e0c9e73))
+ - **FIX**(dynamite): Escape HTML in docs. ([063730f4](https://github.com/nextcloud/neon/commit/063730f448be7c9806183a844ad3fab4275e8580))
+ - **FIX**(dynamite): Escape wireNames. ([5915c570](https://github.com/nextcloud/neon/commit/5915c57035879db20ba7700924dffaf70255b549))
```

Comment:
This looks wrong to me, the new version should be `0.5.1` since there were fixes

Reply:
Looks right to me.
With 0.5.0 being < 1.0.0 we are in the range where we'd use the `+`

Reply:
True I overlooked that, but it should still increase the minor version.

Reply:
FFS I forgot the Dart convention just shifts everything down by one place. Then this is actually correct :woman_facepalming: